### PR TITLE
Updated handling of test config so it can be used with cloud

### PR DESCRIPTION
**Pull request by andy-faunadb**

### CODE REVIEW

File: test/util.js [hunk 1]
```diff
@@ -12,18 +13,20 @@ try {
     domain: env.FAUNA_DOMAIN,
     scheme: env.FAUNA_SCHEME,
     port: env.FAUNA_PORT,
-    rootKey: env.FAUNA_ROOT_KEY
+    auth: {user: env.FAUNA_ROOT_KEY}
```
Comment (eaceaser) [inline test/util.js]: how does one specify the user/pass for cloud?

Comment (andy-faunadb) [inline test/util.js]: `{user: 'me@example.com', pass: 'swordfish'}`

Comment (eaceaser) [inline test/util.js]: sorry; what i mean is, how would you do it using the env vars?

File: test/util.js [hunk 2]
```diff
@@ -1,29 +1,42 @@
...
+    auth: parseAuth(env.FAUNA_ROOT_KEY)
   }
 }
 
+function parseAuth(authStr) {
+  // Split on first ':' to get user:pass
+  const parts = authStr.split(':')
+  const user = parts.shift()
+  const pass = parts.join(':')
+  console.log({user, pass})
```
Comment (eaceaser) [inline test/util.js]: debug stuff

**Comment by eaceaser** [comment]
lgtm sans debug line

**Pull request merged**